Cargo.toml

```
@@ -10,5 +10,5 @@ repository = "https://github.com/georust/topojson"
 [dependencies]
 serde = "~1.0"
 serde_json = "~1.0"
-geojson = ">=0.16.0, <0.24.0"
+geojson = ">=0.16.0, <=0.24.0"
```

```suggestion
geojson = ">=0.16.0, <0.25"
```

I don't know what `<=0.24.0` does for `0.24.1`.

---

Ah, good catch, thanks!